src/matrix-utils.ts

```diff
@@ -161,16 +180,16 @@ export async function createRoom(client, name, isPtt = false) {
   console.log({ isPtt });
 
   await client.createGroupCall(
-    room_id,
+    createRoomResult.room_id,
     isPtt ? GroupCallType.Voice : GroupCallType.Video,
     isPtt,
     GroupCallIntent.Prompt
   );
 
-  return room_alias || room_id;
+  return createRoomResult.room_id;
```

Comment:
What's the reason for this change? I think this means you'll now no longer get the room alias in the URL when creating a room

Reply:
Good catch - I took this out as it wasn't in the types (it's unspecced) but yeah, it does need the room alias. I'm not sure why it fell back to the ID though, so have removed that.